src/components/features/Questions/Question-UI/QuestionModules/MathQuestion.tsx

```
@@ -52,29 +228,135 @@ const MathQuestion: React.FC<{ onAnswerSubmit: (answer: Answers) => void }> = ({
+                  }
+                </span>
+                <span>
+                  Score: {correctCount}/{userAnswered ? questionIndex + 1 : questionIndex} correct
+                </span>
+              </div>
+              
+              {/* Stimulus section */}
+              {stimulus && (
+                <div 
+                  className="mb-5 text-lg question" 
+                  dangerouslySetInnerHTML={{__html: stimulus}}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)